package.js

```diff
@@ -1,22 +1,26 @@
 Package.describe({
   summary: 'Style with attitude. Sass and SCSS support for Meteor.js.',
-  version: '4.15.1',
+  version: '4.16.0',
   git: 'https://github.com/Meteor-Community-Packages/meteor-scss.git',
   name: 'fourseven:scss',
 });
 
 Package.registerBuildPlugin({
   name: 'compileScssBatch',
-  use: ['caching-compiler@1.2.2 || 2.0.0', 'ecmascript@0.15.1'],
+  use: [
+    'caching-compiler',
+    'ecmascript'
+  ],
   sources: ['plugin/compile-scss.js'],
   npmDependencies: {
-    'node-sass': '4.14.1',
-  },
-});
+    "node-sass": '9.0.0',
```

Comment:
I use a JavaScript-only implementation of [sass](https://www.npmjs.com/package/sass) in my package, the main benefit is that it runs in any CPU architecture.

Reply:
Perhaps `node-sass` has implemented arm64 support since v4, it's worth checking. But I would still recommend moving to a JavaScript-only implementation for future proofing.

Reply:
That's interesting, I'll try to add in your fork to our application and see if it works. :+1:

Reply:
This is what happened after adding your package. It renders a blank page with no errors whatsoever either in the web console or server terminal. 

![Leo2](https://github.com/Meteor-Community-Packages/meteor-scss/assets/6644545/01f86b34-78f9-4867-bd59-fdce3203dadf)
![leo_1](https://github.com/Meteor-Community-Packages/meteor-scss/assets/6644545/c2aeac7e-3cce-4bfc-b644-ca3ee3c94a74)

Reply:
That's weird, the only thing "different" is package imports? Which package are you trying to import?

Reply:
Was it running before adding it? Any chance of this being caused by another package?